sites/page-details/index.json-ld

```diff
@@ -82,5 +83,346 @@
 				"path": "deprecated/page-details-v1-fr.html"
 			}
 		]
-	}
+	},
+	"dependencies": [
+		"Page feedback area",
+		"Share this page",
+		"Date modified"
+	],
+	"a11yGuidance": "no accessibility guidance",
+	"variations": [
+		{
+			"name": {
+				"en": "Page details with Page feedback tool",
+				"fr": "Détails de la page avec Outil de rétroaction sur la page"
+			},
+			"status": "stable",
+			"description": {
+				"en": "This component displays the Page details section in a <code>&lt;footer&gt;</code> tag. It contains the following components: Page feedback tool, Share this page, and Date modified.",
```

Comment:
This variation code sample are only showing an implement with the "section" not with the "footer". We probably need to re-write this description.